src/userForm/UserForm.tsx

```diff
@@ -1,71 +1,94 @@
 // Render Prop
 import { Field, Form, Formik, FormikValues } from "formik";
+import { useState } from "react";
+import CustomSelect, { Option } from "./CustomSelect";
 
-const UserForm = () => (
-  <div
-    id="pageContainer"
-    className="h-screen w-full flex flex-col justify-center items-center"
-  >
-    <div className="w-[600px] h-[550px] bg-primary_blue absolute -z-20 rotate-45 rounded-badge left-[-18.5rem] top-[-15rem] flex-0"></div>
-    <div className="w-[300px] h-[150px] border-solid border-12 border-secondary_orange absolute rounded-badge -z-10 left-[6rem] top-[-8rem] rotate-45 flex-0"></div>
-    <div className="w-[600px] h-[550px] bg-primary_blue absolute -z-20 rotate-45 rounded-badge right-[-15rem] bottom-[-25rem] flex-0"></div>
-    <div className="w-[300px] h-[150px] border-solid border-12 border-secondary_orange absolute rounded-badge -z-10 right-[-14rem] bottom-[4rem] rotate-45 flex-0"></div>
+function UserForm() {
+  //this is just mock data to be populated later
+  const options: Option[] = [
+    { value: "Username", label: "Username 1" },
+    { value: "Example", label: "Example 2" },
+    { value: "Here", label: "Here 3" },
+  ];
+
+  const [selectedValue, setSelectedValue] = useState<Option | null>(null);
+
+  return (
     <div
-      className="flex flex-col bg-off_white w-4/5 h-2/5 rounded-2xl shadow-3xl px-8 py-7 justify-around"
-      id="formContainer"
+      id="pageContainer"
+      className="h-screen w-full flex flex-col justify-center items-center overflow-hidden overscroll-none"
     >
-      <div className="flex flex-col gap-1.5">
-        <h1 className="text-dark_text text-2xl">Find a User</h1>
-        <h2 className="text-ligher_text text-sm">
-          Enter the Github username of the account you would like to view.
-        </h2>
+      <div className="h-screen w-screen overflow-hidden absolute -z-20">
+        <div className="w-[600px] h-[550px] bg-primary_blue absolute -z-20 rotate-45 rounded-badge left-[-18.5rem] top-[-15rem] flex-0"></div>
```

Comment:
So while the custom values are correct, I would suggest using either `w-[rem]` value or builtin tailwind w-values.